### PR TITLE
Turn on Atlas security

### LINKED ISSUE #99: Deployment is able to define custom users
- Config Web API to work only with Token and authorize using it.
- New schema and table for users and passwords. (user mgt)
- New role for researcher users, (hash password received from UI)
- Add relevant Administrator permissions

**Acceptance Criteria**
User Admin and User researcher can authenticate to Atlas UI using basic auth. 
Admin has all permissions

### PR DESCRIPTION
**Pull request by tamirkamara**
Turns on Atlas basic security with the local db.
Fixes #99.

### CODE REVIEW

File: infra/main.bicep [hunk 1]
```diff
@@ -147,3 +138,46 @@ module atlasUI 'ohdsi_atlas_ui.bicep' = {
 }
 
 output ohdsiWebapiUrl string = ohdsiWebApiWebapp.outputs.ohdsiWebapiUrl
+
+
+resource deploymentAtlasSecurity1 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
```
Comment (anatbal) [inline infra/main.bicep:143]:
```suggestion
resource deploymentAtlasSecurity 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
```

File: infra/ohdsi_webapi.bicep [hunk 2]
```diff
@@ -30,114 +29,157 @@ resource webApp 'Microsoft.Web/sites@2022-03-01' = {
...
+          name: 'DOCKER_REGISTRY_SERVER_URL'
+          value: dockerRegistryServer
+        }
+        {
+          name: 'DATASOURCE_DRIVERCLASSNAME'
+          value: 'org.postgresql.Driver'
+        }
+        {
+          name: 'DATASOURCE_OHDSI_SCHEMA'
+          value: postgresWebApiSchemaName
```
Comment (anatbal) [inline infra/ohdsi_webapi.bicep:44]: Do you want to make it as hardcoded 'webapi' as we discussed or should it be done in a separate PR?

File: infra/sql/atlas_security.sql [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+DROP TABLE IF EXISTS webapi_security.security;
+
+CREATE TABLE webapi_security.security
+(
+    email character varying(255),
+    password character varying(255)
+);
+
+insert into webapi_security.security (email,password) values ('admin', '$2a$10$opEKwT32fEvoPfSbzE1Rx.p8QsCG0KryiA7VEguLP/V0M62aho6mC');
```
Comment (anatbal) [inline infra/sql/atlas_security.sql:11]: Is this hash means the password is hardcoded?

File: infra/ohdsi_webapi.bicep (hunk 2 above)
Comment (tamirkamara) [reply infra/ohdsi_webapi.bicep:44]: I didn't change it yet as I'm not 100% sure and it can wait.

File: infra/sql/atlas_security.sql (hunk 3 above)
Comment (tamirkamara) [reply infra/sql/atlas_security.sql:11]: yes, for now.

**Review by anatbal — APPROVED** [review]
